test/conftest.py

```diff
@@ -1,4 +1,79 @@
 import pytest
+import psycopg
+from dotenv import dotenv_values
+
+from src.application import app
+from tools.ConnectionManager import ConnectionManager
+from fastapi.testclient import TestClient
+import os
+
+
+@pytest.fixture(scope="session")
+def db_connection():
+    """
+    Establishes odbc connection for the whole session
+    :return: pyodbc connection class
+    """
+    env_path = "db/.env"
+    if os.path.exists(env_path):
+        env_values = dotenv_values(env_path)
+        env_variables = ["POSTGRES_USER", "POSTGRES_DB", "POSTGRES_PASSWORD"]
+        for env_variable in env_variables:
+            os.environ[env_variable] = env_values.get(env_variable)
+
+    db_connection_config = {
+        "host": "localhost",
+        "dbname": os.getenv("POSTGRES_DB"),
+        "user": os.getenv("POSTGRES_USER"),
+        "password": os.getenv("POSTGRES_PASSWORD"),
+    }
+    testdb_connection_config = {
+        "host": "localhost",
+        "dbname": "testdb",
+        "user": os.getenv("POSTGRES_USER"),
+        "password": os.getenv("POSTGRES_PASSWORD"),
+    }
+
+    with psycopg.connect(**db_connection_config, autocommit=True) as conn:
+        with conn.cursor() as cur:
+            cur.execute("DROP DATABASE IF EXISTS testdb")
+            cur.execute("CREATE DATABASE testdb")
```

Comment:
why do you need to do this on `db_connection` instead of `test_connection`? i assumed you have a separate container for the test database, and then you make sure that you reset it before (or after) all tests are done - but maybe I'm misunderstanding the solution :)

Reply:
In this branch I have a test database in the same container as the "real" database. You didn't ask for tests running on a different container until the next step.
I do it here because I do it once at the start of the tests to initialize an empty database for all tests, and I have all the fixtures in conftest.

Reply:
ok then :)